contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -2430,7 +2428,7 @@ sp_addrolemember(PG_FUNCTION_ARGS)
 					 errmsg("Cannot make a role a member of itself.")));
 
 		/* Map the logical member name to its physical name in the database. */
-		physical_member_name = get_physical_user_name(get_cur_db_name(), lowercase_membername, false);
+		physical_member_name = get_physical_user_name(get_cur_db_name(), lowercase_membername, false, true);
```

Comment:
pfree is needed? plz check all the call of get_physical_user_name

Reply:
Done